packages/mui-joy/src/CircularProgress/CircularProgress.tsx

```diff
@@ -31,7 +31,7 @@ const useUtilityClasses = (ownerState: CircularProgressOwnerState) => {
   const slots = {
     root: [
       'root',
-      determinate && 'indeterminate',
+      determinate ? 'determinate' : 'indeterminate',
```

Comment:
@hbjORbj I don't think we need to change this. From my understanding, the classes are for custom styling, so you are already able to customize both states:

```css
// custom indeterminate state
background-color: ...;

.JoyCircularProgress-indeterminate {
  background-color: ...;
}
```

Reply:
Also, Material UI's circular progress does not introduce `indeterminate` class. I think we can drop this PR.

Reply:
As we discussed on Slack, I will apply `determinate && 'determinate',` only.